test/framework/functional/elasticsearch.go

```diff
@@ -20,54 +19,26 @@ func (f *CollectorFunctionalFramework) GetLogsFromElasticSearch(outputName strin
 	if !ok {
 		return "", fmt.Errorf(fmt.Sprintf("can't find log of type %s", outputLogType))
 	}
-	return f.GetLogsFromElasticSearchIndex(outputName, index)
+	return f.GetLogsFromElasticSearchIndex(f.Test.NS.Name, f.Pod.Name, outputName, index)
 }
 
-func (f *CollectorFunctionalFramework) GetLogsFromElasticSearchIndex(outputName string, index string) (result string, err error) {
+func (f *CollectorFunctionalFramework) GetLogsFromElasticSearchIndex(namespaceName string, podName string, outputName string, index string) (result string, err error) {
```

Comment:
The method arg does not need to be changed.  These can be obtained from 'f' as part of the framework